scene/main/window.cpp

```diff
@@ -88,6 +88,10 @@ Size2i Window::get_size() const {
 	return size;
 }
 
+void Window::reset_size() {
+	set_size(Size2i(-1, -1));
```

Comment:
I'd suggest using `Size2i()`, it should be equivalent and negative sizes are a bit weirder to reason about (even if the result is the same as it will be directly overridden).